projects/services/storage-service-typescript/src/server.ts

```diff
@@ -15,14 +16,22 @@ export default function(nodeTopology: any, env: any) {
     const peers = topologyPeers(nodeTopology.peers);
     // TODO: this is very fragile - refactor to make this less hardcoded
     const blockStorageDBPath = BLOCK_STORAGE_DB_PATH || path.resolve(path.join("../../../db", NODE_ENV || "development") + "blocks.db");
+
+    const verifySignature = _.lowerCase(BLOCK_STORAGE_VERIFY_SIGNATURES) === "true";
```

Comment:
I think that the default should be that signatures are verified, and they can be disabled under certain circumstances

Reply:
The verification is enabled during deployment, but if we enable it by default we'll have to do additional work around e2e.